### PR TITLE
Update common-marvelmind.rst

### PR DESCRIPTION
**Pull request by smoker771**
We propose these changes, because previous version of the article is not actual, the recommendations were obsolete and don't work correctly with last versions of the Ardupilot and Marvelmind software. The proposed version includes link to tested sets of settings which we will continue update in future for new versions of software

### CODE REVIEW

File: common/source/docs/common-marvelmind.rst [hunk 1]
```diff
@@ -17,43 +17,29 @@ See the `user manual <https://marvelmind.com/pics/marvelmind_navigation_system_m
...
 - Connect the Autopilot to a ground station
 - Click on 'freeze map' on the dashboard
-- After about 1 minute the vehicle's position should jump to the latitude, longitude you input during the configuration step (above)
 - Check that the vehicle's position is relatively stable (i.e. moving around less than one meter)
 - Walk the vehicle around between the beacons and ensure that its position on the map updates correctly
 
-DataFlash logging
-=================
 
-The distance to the beacons can be found in the dataflash log's BCN message's D0, D1, D2, D3 fields.
```
Comment (Hwurzburg) [inline common/source/docs/common-marvelmind.rst:59]: why doesnt the beacon logging work like it did before?

Comment (smoker771) [reply common/source/docs/common-marvelmind.rst:59]: New version of the article recommends to setup NMEA0183 protocol mode. Distances to the beacons are not available in this mode. Previous version was based on Marvelmind proprietary protocol, this protocol supports distances. But Marvelmind protocol seems to be not working with Ardupilot in several last versions of software, this was a reason for the update of the article

Comment (Hwurzburg) [reply common/source/docs/common-marvelmind.rst:59]: you should raise an issue about that so that it gets fixed

**Comment by rmackay9** [comment]
This is a very different setup and means that the marvelmind acts like a GPS which is OK but removes the ability to do GPS-to-NonGPS transitions.

I thought @amilcarlucas knew the most about the marvelmind but I see that @peterbarker actually [wrote the driver here](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Beacon/AP_Beacon_Marvelmind.h)

@smoker771 do you work at Marvelmind?

**Comment by smoker771** [comment]
Yes, I'm working in Marvelmind. 
We really appreciate this article. This article was valid some time ago, but now it is not valid for some reasons. Many Ardupilot users try to use it without success, ask us to help and we recommend switching to NMEA mode. Reason of changes to the article was to help Ardupilot users integrate with Marvelmind system  successfully. 
Thank you

**Review by rmackay9 — APPROVED** [review]

File: common/source/docs/common-marvelmind.rst [hunk 2]
```diff
@@ -17,43 +17,29 @@ See the `user manual <https://marvelmind.com/pics/marvelmind_navigation_system_m
...
-- Set :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>` to the heading from the origin beacon to the 2nd beacon.  One way to capture this value is to stand at the origin holding the vehicle so that it's nose points towards the second beacon.  Read the vehicle's heading from the HUD and enter this value into :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>`
-- Set :ref:`GPS1_TYPE <GPS1_TYPE>` to 0 to disable the GPS
-- Set :ref:`ARMING_CHECK <ARMING_CHECK>` to -9 to disable the GPS arming check
-- Set :ref:`SERIAL1_BAUD <SERIAL1_BAUD>` to 115 to set telemetry1's baud rate to 115200 (if using Telem2 set :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` instead)
-- Set :ref:`SERIAL1_PROTOCOL <SERIAL1_PROTOCOL>` to 13 to enable receiving data from the marvelmind (If using Telem2 set :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` instead)
-- Set :ref:`BRD_SER1_RTSCTS <BRD_SER1_RTSCTS>` to 0 to ensure telem1 does not use flow control (If using Telem2 set :ref:`BRD_SER2_RTSCTS <BRD_SER2_RTSCTS>` instead, not required if using Serial4/5)
+- Here is a package with tested param files for the Autopilot and Marvelmind hedge: <https://marvelmind.com/pics/marvelmind_ardupilot_settings.zip>
```
Comment (rmackay9) [inline common/source/docs/common-marvelmind.rst:32]: I've approved but this line looks a little odd.  I think there should be some single quotes around it.

I also think that we probably need to set the [GPS1_TYPE](https://ardupilot.org/copter/docs/parameters.html#gps1-type-gps-type) = 5 (NMEA)

Comment (smoker771) [reply common/source/docs/common-marvelmind.rst:32]: Thank you for approving. 
We included all settings into the "param" files available by the link and "GPS_TYPE = 5" is also set there.
I will make an additional pull request to fix this odd line and some small changes in the design.

**Pull request merged**